### PR TITLE
Runs cross platform build to ensure all the checks pass

### PR DESCRIPTION
**Pull request by navidshaikh**
- Test PR for #157
 /hold

### CODE REVIEW

File: openshift/e2e-tests-openshift.sh [hunk 1]
```diff
@@ -196,7 +197,21 @@ EOF
 }
 
 function deploy_serverless_operator(){
-  oc apply -f openshift/serverless/operator-install.yaml
+  git clone https://github.com/openshift-knative/serverless-operator.git /tmp/serverless-operator
```
Comment (rhuss) [inline openshift/e2e-tests-openshift.sh]: Is it guaranteed that there is no `/tmp/serverless-operator` ? Better use `mktemp` for creating a temporary name and clean it up afterwards (store it in a var)

File: openshift/e2e-tests-openshift.sh [hunk 2]
```diff
@@ -196,7 +197,21 @@ EOF
...
+  /tmp/serverless-operator/hack/catalog.sh | oc apply -n $CS_NS -f -
+  cat <<-EOF | kubectl apply -f -
```
Comment (rhuss) [inline openshift/e2e-tests-openshift.sh]: Not sure you need this `-` in `<<-EOF`. I always use it without.

File: openshift/e2e-tests-openshift.sh (hunk 1 above)
Comment (navidshaikh) [reply openshift/e2e-tests-openshift.sh]: IIUC, these operations are performed in a test pod which is a fresh one per PR test and goes away after the tests.

File: openshift/e2e-tests-openshift.sh (hunk 2 above)
Comment (navidshaikh) [reply openshift/e2e-tests-openshift.sh]: I borrowed this as is from https://github.com/openshift-knative/serverless-operator/blob/master/README.md#create-a-subscription from @jcrossley3 , may be he has something to add here.

**Pull request merged**